docs/MagicNumberDetector.md

```diff
@@ -0,0 +1,122 @@
+# Magic Number Detector
+
+This Command help to find Magic Numbers in your code or in specific php language constructs (like just checking switch cases etc.). It can also ignore `1`, `0` and `-1` (if you want to). 
```

Comment:
```suggestion
This command helps to find Magic Numbers in your code or in specific php language constructs (like just checking switch cases etc.). It can also ignore `1`, `0` and `-1` (if you want to). 
```